aries-site/src/examples/templates/forms/PayExample.js

```diff
@@ -105,6 +105,19 @@ const FormContainer = ({ ...rest }) => {
   );
 };
 
+const Error = ({ textError, ...rest }) => {
+  return (
+    <Box {...rest} align="center" gap="xsmall" direction="row">
+      <CircleAlert size="small" />
+      <Text size="xsmall">{textError}</Text>
+    </Box>
+  );
+};
```

Comment:
Move ...rest to be last property so that it ensures whatever is supplied to rest gets applied and not overwritten.

```suggestion
const Error = ({ textError, ...rest }) => {
  return (
    <Box align="center" gap="xsmall" direction="row" {...rest} >
      <CircleAlert size="small" />
      <Text size="xsmall">{textError}</Text>
    </Box>
  );
};
```